src/test/java/org/testcontainers/containers/TarantoolCartridgePortMappingTest.java

```diff
@@ -0,0 +1,104 @@
+package org.testcontainers.containers;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.Duration;
+import java.util.concurrent.ThreadLocalRandom;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.Test;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.junit.jupiter.Container;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TarantoolCartridgePortMappingTest {
+
+    @Container
+    private final static TarantoolCartridgeContainer container = new TarantoolCartridgeContainer(
+            "Dockerfile",
+            "mapping-ports-container",
+            "cartridge/instances.yml",
+            "cartridge/replicasets.yml")
+            .withEnv(TarantoolCartridgeContainer.ENV_TARANTOOL_CLUSTER_COOKIE, "secret")
+            .withRouterUsername("admin")
+            .withRouterPassword("secret")
+            .withStartupTimeout(Duration.ofMinutes(5))
+            .withLogConsumer(new Slf4jLogConsumer(
+                    LoggerFactory.getLogger(TarantoolCartridgeBootstrapFromYamlTest.class)));
+
+    @Test
+    void portMappingTest() throws IOException, URISyntaxException {
+
+        final int httpPortToFirstRouter = 8081;
+        final int httpPortToSecondRouter = 8082;
+        final int portToFirstRouter = 3301;
+        final int portToSecondRouter = 3302;
+        final String host = "localhost";
+
+        container.addExposedPorts(httpPortToFirstRouter, httpPortToSecondRouter, portToFirstRouter, portToSecondRouter);
+        container.start();
+
+        URI uri = new URI("http", null, host, container.getMappedPort(httpPortToFirstRouter), null,
```

Comment:
Use more specific name for variable.
It's bad practice to use generic names for a variables.

Reply:
Fixed